user_tools/tests/spark_rapids_tools_e2e/resources/scripts/hdfs/setup_hdfs.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: ./setup_hdfs.sh [_HDFS_SHOULD_RUN]
```

Comment:
add description to these scripts unless its obvious... how many nodes does this setup, how much space, what directories does it use, etc.

Reply:
Added description about the replication factor, disk space and temp directory used